modules/react/card/lib/Card.tsx

```diff
@@ -60,28 +58,21 @@ const Box = styled('div', {
   ({height}) => height && {height}
 );
 
-const Header = styled('h3')(type.h3, {
-  marginBottom: space.m,
-  marginTop: 0,
-});
-
-const Body = styled('div')(type.body);
-
-export default class Card extends React.Component<CardProps> {
-  public render() {
-    const {
-      depth = depthValues[2],
-      padding = space.l,
-      heading,
-      headingId,
-      ...elemProps
-    } = this.props;
-
+export const Card = createComponent('div')({
+  displayName: 'Card',
+  Component: (
+    {depth = depthValues[2], children, padding = space.l, ...elemProps}: CardProps,
+    ref,
+    Element
+  ) => {
     return (
-      <Box {...elemProps} depth={depth} padding={padding}>
-        {heading && <Header id={headingId}>{heading}</Header>}
-        <Body>{this.props.children}</Body>
+      <Box ref={ref} as={Element} depth={depth} padding={padding} {...elemProps}>
+        {children}
       </Box>
     );
-  }
-}
+  },
+  subComponents: {
+    Header: CardHeader,
```

Comment:
Any reason why you chose `CardHeader` instead of `CardHeading`? Technically, `heading` is the correct term and it's what was used for the prop before.

Reply:
Good point. I like "Heading" better. I had issues with using both "Heading" and "Header", but "Heading" is more accurate. I'll update.